spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/ModuleSerializable.scala

```diff
@@ -142,6 +143,7 @@ trait ModuleSerializable extends Loadable with Savable{
         clsMirror.reflect(compnInst)
     }
 
+    // Todo: to be replaced with ReflectionUtils.reflect
```

Comment:
Remove this line as we already use the Util  ?

Reply:
No. This piece of code is duplicated with RefelctionUtils.reflect due to minor differences.
Would like to make a separate PR for the refactor.